compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -332,7 +332,7 @@ OMR::ResolvedMethodSymbol::getLogicalParameterList(TR::Compilation *comp)
    {
    if (comp->getMethodSymbol() == self())
       {
-      List<TR::ParameterSymbol>* l = comp->cg()->getLinkage()->getMainBodyLogicalParameterList();
+      List<TR::ParameterSymbol>* l = NULL;
```

Comment:
I think there's further folding that could be done here, given that literally the next check is `if (l == null)`